components/speedreader/speedreader_rewriter_service.cc

```diff
@@ -1,21 +1,25 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/components/speedreader/speedreader_rewriter_service.h"
+#include "brave/components/speedreader/speedreader_util.h"
 
```

Comment:
this should go to the common block of includes, only our own one sits on the top